inc/checks/class-language-update.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace runcommand\Doctor\Checks;
+
+use WP_CLI;
+
+/**
+ * Warns when there are language updates available.
+ */
+class Language_Update extends Check {
+
+	public function run() {
+		ob_start();
+		WP_CLI::run_command( array( 'language', 'core', 'list' ), array( 'format' => 'json' ) );
+		$ret = ob_get_clean();
+		$languages = ! empty( $ret ) ? json_decode( $ret, true ) : array();
+		$update_count = 0;
+		foreach ( $languages as $language ) {
+			if ( 'available' === $language['update'] ) {
+				$update_count++;
+			}
+		}
+
+		if ( 1 === $update_count ) {
```

Comment:
In general, we should prefer to avoid else clauses whenever possible, to keep code complexity in check.

Here, I'd write the conditions in the following way:

```php
if ( ! $update_count ) {
   $this->set_status( 'success' );
   $this->set_message( 'Languages are up to date.' );
   return;
}

$message = 1 === $update_count
   ? '1 language has an update available.'
   : "{$update_count} languages have updates available.";

$this->set_status( 'warning' );
$this->set_message( $message );
```